drivers/syslog/vsyslog.c

```diff
@@ -128,7 +130,7 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
 #if defined(CONFIG_SYSLOG_COLOR_OUTPUT)
   /* Reset the terminal style. */
 
-  ret = lib_sprintf(&stream.public, "\e[0m");
+  offset += sprintf(offset, "\e[0m");
```

Comment:
why can we boost syslog speed by changing lib_sprintf to sprintf?

Reply:
lib_sprintf directly calls syslog_putc to output character by character
sprintf can save the formatted string to the buffer, and then call syslog_write once to output.

It actually reduces the number of calls to syslog_putc

Reply:
could you add the perf number in the commit message?

Reply:
performance test data can be given tomorrow